<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>541</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QPushButton" name="RecommendPage">
      <property name="text">
       <string>推荐界面</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="OnlinePeople">
      <property name="text">
       <string>用户在线管理</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="dateLookUp">
      <property name="text">
       <string>数据查看</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>411</width>
          <height>232</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="4">
          <widget class="QPushButton" name="DeleteBt">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="PartTimeJobradioButton">
           <property name="text">
            <string>兼职</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="FarmRadioButton">
           <property name="text">
            <string>农产品</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="5">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_2"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="WritrEditLine"/>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="OkBt">
           <property name="text">
            <string>确认添加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>输入单号：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>物品交易</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>农业推荐：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>物换物推荐：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>兼职推荐：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>351</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>在线用户：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>强制下线</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QListWidget" name="OnlinePeoplelistWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="gridLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>431</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>兼职信息</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="LookUpFarmProduct">
           <property name="text">
            <string>农业</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_4">
           <property name="text">
            <string>商品交换</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QTableView" name="DatabaseTableView"/>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="UpdateDateBase">
           <property name="text">
            <string>刷新查看</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="AddTheList">
           <property name="text">
            <string>添加到推荐界面</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
